index.html

```diff
@@ -234,46 +246,25 @@
-                                <td class="form-td"><input type="text" placeholder="Last Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
+                    <form action="https://formspree.io/f/moqzdnpo" method="post">
+                        <input type="text" name="first-name" placeholder="First Name">
+                        <input type="text" name="last-name" placeholder="Last Name">
+                        <input id="email" name="e-mail" type="email" placeholder="E mail address">
+                        <td colspan="2" name="message" class="form-td"><textarea>Typing something</textarea>
+                        <button class="see-project-btn">Get in touch</button>
                     </form>
                 </div>
                 <div class="touch-class3">
-                    <form>
-                        <table class="form-table">
-                            <tr class="form-tr couple-tr">
-                                <td class="form-td"><input type="text" placeholder="Full Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td align="center"><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
+                    <form class="mob_form" action="https://formspree.io/f/moqzdnpo" method="post">
+                        <input type="text" name="full-name" maxlength="30" required 
+                         placeholder="Full Name">
+                        <input id="email" name="e-mail" required type="email" placeholder="Email address">
+                        <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+                        <button class="see-project-btn mob_btn">Get in touch</button>
                     </form>
```

Comment:
- [ ] Please, kindly ensure that **all** your design specifications (padding, margin, font-size, font-weight, border-radius and so on) align with that of Figma as much as possible, So as to have a pixel perfect design as you're going to soon be doing in your future job 😀.

- Your design
<img width="161" alt="wrong1" src="https://user-images.githubusercontent.com/41163571/220488655-25c9dd2f-5cae-4e2f-92ea-314dec724e10.png">


- Figma design
<img width="186" alt="right1" src="https://user-images.githubusercontent.com/41163571/220487862-e98308dc-9214-4d40-a822-2cab63b13a91.png">

And please, ensure that elements don't overlap. 😀

Reply:
![image](https://user-images.githubusercontent.com/54537654/220782759-9b9acb64-ca19-4d48-89f9-b7d703353786.png)
I've now ensured that the mentioned problem don't appear again. Here is the demonstration